tests/recipes/test_eleuther_eval.py

```diff
@@ -0,0 +1,98 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import builtins
+import runpy
+
+import sys
+from pathlib import Path
+
+import pytest
+
+import torchtune
+from tests.common import TUNE_PATH
+from tests.recipes.utils import fetch_ckpt_model_path, llama2_small_test_ckpt
+from torchtune import models
+
+pkg_path = Path(torchtune.__file__).parent.parent.absolute()
+EVAL_CONFIG_PATH = Path.joinpath(
+    pkg_path, "recipes", "configs", "llama2_eleuther_eval.yaml"
+)
+
+models.small_test_ckpt_tune = llama2_small_test_ckpt
```

Comment:
Merge with testing PR changes and u will live a happy and fulfilling life

Reply:
Merge bad, rebase good.

Reply:
wait yall don't use rebase?